src/buttons/button-radio-group.directive.ts

```diff
@@ -43,4 +45,16 @@ export class ButtonRadioGroupDirective implements ControlValueAccessor {
   registerOnTouched(fn: () => {}): void {
     this.onTouched = fn;
   }
+
+  setDisabledState(disabled: boolean): void {
+    const radioButtons = this.el.nativeElement.querySelectorAll('[btnRadio]');
```

Comment:
please, use ViewChildren instead

Reply:
@valorkin I fixed this comment. Could you please recheck my PR?